apps/aurora-portal/src/client/routes/_auth/accounts/$accountId/projects/$projectId.tsx

```diff
@@ -24,7 +24,7 @@ function RouteComponent() {
       <div className="w-full flex">
         <ProjectSubNavigation />
       </div>
-      <div className="py-4 pl-4 bg-theme-global-bg h-full">
+      <div className="py-4 pl-4 bg-theme-background-lvl-0  h-full">
```

Comment:
It shouldn't be necessary to set a background color here. If I understand correctly, this is rendered inside the `AppContent` in the `AppShell`. The layout provided by the `AppShell` takes care of rendering the necessary parent elements with correct background colors. You shouldn't set a background color (unless necessary) because you would overwrite any future changes made to the theme. 
```suggestion
      <div className="py-4 pl-4 h-full">
```

Reply:
Problem is because of that in light-theme it rendered lvl-1 instead of lvl-0 on some pages. dark-theme was ok.